Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
I think the question makes sense: it's asking what is the range of WeekOfMonth values within a year. Normally (always?) it can range from 1 to 6.

---
In reply to: [74356593](https://github.com/Microsoft/WinObjC/pull/844#discussion_r74356593) [](ancestors = 74356593,74354002,74342094,74336418)